### PR TITLE
Receiving events via Kafka and Publishing to WebSocket

### PR DESCRIPTION
**Pull request by OshiWindsor**
Added Receiving events via Kafka and Publishing to WebSocket tutorial

### CODE REVIEW

File: en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# Recieving XML events via Kafka and Publishing to WebSocket
+
+## Introduction
+
+The Streaming Integrator can consume events from a Kafka topic and publish those events to a WebSocket  in a streaming manner.
```
Comment (samgnaniah) [inline en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md:5]:
```suggestion
The Streaming Integrator can consume events from a Kafka topic and publish those events to a WebSocket in a streaming manner.
```

Shall we expand what we mean by "streaming manner"?

File: en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            * For Windows:
+                ```
+                {WSO2SIHome}/bin/jartobundle.bat <{Source} Directory Path> <{Destination} Directory Path>
+                ```
+            * For Linux:
+                ```
+                sh {WSO2SIHome}/bin/jartobundle.sh <{Source} Directory Path> <{Destination} Directory Path>
+                ```
+                
+5. Add the OSGI converted kafka libs from the `Destination` directory to the `<SI_HOME>/lib` directory.
```
Comment (samgnaniah) [inline en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md:46]:
```suggestion
5. Add the OSGI converted Kafka libs from the `Destination` directory to the `<SI_HOME>/lib` directory.
```

File: en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+6. Add the original Kafka libs from `Source` to `<SIHome>/samples/sample-clients/lib`.
+
+## Consuming data from Kafka
+
+### Step 1: Start Kafka
+
+1. Navigate to the `<KAFKA_HOME>` directory and start a zookeeper node by issuing the following command.
```
Comment (samgnaniah) [inline en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md:54]:
```suggestion
1. Navigate to the `<KAFKA_HOME>` directory and start a Zookeeper node by issuing the following command.
```

File: en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+```
+
+#### Executing the Sample:
+
+1. Open a terminal and navigate to `<SIHome>/samples/sample-clients/kafka-producer` and run "ant" command as follows:
+```
+ant -Dtype=xml -DtopicName=kafka_sample_topic
+```
+
+You can limit the number of events as follows as well,
```
Comment (samgnaniah) [inline en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md:117]: Does the number formatting break here?

File: en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+```
+ant -Dtype=xml -DnoOfEventsToSend=5 -DtopicName=kafka_sample_topic
+```
+
+2. Open a terminal and navigate to the `<SIHome>/samples/sample-clients/websocket-receiver` directory and run the ant command.
```
Comment (samgnaniah) [inline en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md:122]:
```suggestion
2. Open a terminal and navigate to the `<SIHome>/samples/sample-clients/websocket-receiver` directory and run the `ant` command.
```

File: en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+        * RecieveKafkaPublishWebSocket.siddhi -  Started Successfully!
+        ```
+
+#### Notes:
+
+If you edit this application while it's running, stop the application -> Save -> Start.
+
+## Viewing the Results:
+
+Check the output in the terminal of <SIHome>/samples/sample-clients/websocket-receiver. You will see output similar to the following:
```
Comment (samgnaniah) [inline en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md:142]:
```suggestion
Check the output in the terminal of `<SI_HOME>/samples/sample-clients/websocket-receiver`. You will see output similar to the following: 
```

File: en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md (hunk 4 above)
Comment (OshiWindsor) [reply en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md:117]: No.

**Review by samgnaniah — APPROVED** [review]

File: en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+- Version
+- Description
+- Usages in APIs
+- Other important metadata.
+
+### Step 9: Creation of an API
+
+Create an API from a deployed service by clicking on the **Create API** button in the services listing page.
+
+Provide the details in there for API Name, Context and Version where all fields are mandatory, and click on the **Create API** button.
```
Comment (ramindu90) [inline en/docs/use-cases/streaming-tutorials/receiving-events-via-kafka-and-publishing-to-websocket.md:250]: Add an image of the entered details

**Review by ramindu90 — CHANGES_REQUESTED** [review]

**Review by ramindu90 — APPROVED** [review]

**Pull request merged**